src/main/scala/com/amazon/deequ/analyzers/jdbc/JdbcAnalyzer.scala

```diff
@@ -193,4 +193,13 @@ object Preconditions {
         s"but found ${JDBCType.valueOf(columnDataType).getName} instead!")
     }
   }
+
+  /** Statement has no ; outside of quotation marks (SQL injections)*/
+  def hasNoInjection(statement: Option[String]): Table => Unit = { _ =>
+    val pattern = """("[^"]*"|'[^']*'|[^;'"]*)*"""
+    val st = statement.get
+    if(!st.matches(pattern)){
+      throw new SQLInjectionException(s"In the statement semicolon outside of quotation marks are not allowed to prevent SQL injections")
```

Comment:
this line is to long
```suggestion
      throw new SQLInjectionException(
          s"In the statement semicolon outside of quotation marks are not allowed to prevent SQL injections"
      )
```